edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/BridgeConnection/IMqttBridgeConnector.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
```

Comment:
Super nit: for consistency, can we pls camelCase the folder names (bridgeConnection)